demo/src/plugins/dashboard/components/active-edits/index.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  Table,
+} from '@triniti/admin-ui-plugin/components';
+import delegate from './delegate';
+import selector from './selector';
+import TableRow from './TableRow';
+
+class ActiveEdits extends React.Component {
+    static propTypes = {
```

Comment:
@nanetteCodes i dont think you need to disable eslint for accessToken or collaborationNodes. the first is just a string and the second can be `arrayOf(PropTypes.instanceOf(Message))`

Reply:
updated this